chainntnfs/height_hint_cache.go

```diff
@@ -148,6 +158,10 @@ func (c *HeightHintCache) CommitSpendHint(height uint32,
 // cache for the outpoint.
 func (c *HeightHintCache) QuerySpendHint(spendRequest SpendRequest) (uint32, error) {
 	var hint uint32
+	if c.cfg.HeightHintCacheQueryDisable {
+		Log.Debugf("Ignoring spend height hint for %v (height hint cache query disabled)", spendRequest)
```

Comment:
Nit: lines should wrap at 80 characters with a tab width of 8 spaces. There are a few other cases throughout the diff.